drivers/note/notelog_driver.c

```diff
@@ -319,36 +318,6 @@ void sched_note_spinlock(FAR struct tcb_s *tcb,
 }
 #endif
 
-#ifdef CONFIG_SCHED_INSTRUMENTATION_SYSCALL
-void sched_note_syscall_enter(int nr, int argc, ...)
-{
-  char buf[128];
-  FAR char *p = buf;
-  va_list ap;
-
-  va_start(ap, argc);
-  while (argc-- > 0)
-    {
-      if (argc)
-        {
-          p += sprintf(p, "%#"PRIxPTR", ", va_arg(ap, uintptr_t));
-        }
-      else
-        {
-          p += sprintf(p, "%#"PRIxPTR, va_arg(ap, uintptr_t));
-        }
-    }
-
-  va_end(ap);
-  syslog(LOG_INFO, "%s@%d ENTER %s\n", g_funcnames[nr], nr, buf);
```

Comment:
let's skip the log for syslog/getpid to void the infinite recursive loop instead removing all syscall log?

Reply:
This is not a good idea, it will cause a lot of unnecessary changes, and notelog is not a good way to trace
We should try to use noteram instead of notelog